### PR TITLE
Added D48 compatibility

### PR DESCRIPTION
**Pull request by jeff-o**
Added driver changes, scripts, URDF and meshes for an ethernet-connected D48 PTU.

### CODE REVIEW

File: flir_ptu_driver/src/driver.cpp [hunk 1]
```diff
@@ -176,6 +205,7 @@ float PTU::getPosition(char type)
     ROS_ERROR("Error getting pan-tilt pos");
     return -1;
   }
+  ROS_INFO("pos: %c %s", type, buffer.c_str());
```
Comment (mikepurvis) [inline flir_ptu_driver/src/driver.cpp]: This should be removed.

**Comment by tonybaltovski** [comment]
@TheDash can you review?

**Comment by TheDash** [comment]
There is no way to confirm/deny that this is working currently with their existing firmware as this is over 2 years old. There is no way I can approve this until we get something that either:

1. simulates a d48 firmware interface and has unit tests which test this code against it
2. manually test this with a d48, with logs showing that it can read and write and configure and move this using this driver.

**Comment by civerachb-cpr** [comment]
Given the age of the PR, I'm going to close it.

**Pull request closed without merging**